lib/common/netcdf-check.sh

```diff
@@ -109,20 +95,17 @@ add_checker_signature() {
     local checker_version=`$NETCDF_CHECKER --version`
     local version_number=`echo $checker_version | egrep 'IOOS compliance checker version' | egrep -o '[0-9.]+$'`
 
-    local commit_hash=`_get_checker_commit_hash`
-    local last_updated=`_get_checker_commit_date`
+    local imos_version=$($NETCDF_CHECKER -l | grep ' imos ' | sed -r 's/.*\((.*)\)/\1/')
```

Comment:
Yep, could do, but would still need to grep to extract just the version number from that output.